packages/bulkapi/src/database/LionWebQueries.ts

```diff
@@ -111,6 +119,19 @@ export class LionWebQueries {
         const databaseChunk = await this.context.bulkApiWorker.bulkRetrieve(tbsNodeAndChildIds, "", 0)
         const databaseChunkWrapper = new LionWebJsonChunkWrapper(databaseChunk)
 
+        logger.dbLog("DBChunk " + JSON.stringify(databaseChunk))
+
+        logger.dbLog("ID-2 " + JSON.stringify(databaseChunkWrapper.getNode("ID-2")))
```

Comment:
I am not sure this line should be here

Reply:
I think this may be connected to a test, but this is outside of the test sourceset

Reply:
Shouldn't be there, removed it.